assets/csidriveroperators/aws-ebs/hypershift/mgmt/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ resources:
   - ../../base
 namespace: ${CONTROLPLANE_NAMESPACE}
 patches:
-  - path: 02sa.patch.yaml
+  - path: sa.patch.yaml
     target:
```

Comment:
Purely consistency. Per the commit message

https://github.com/openshift/cluster-storage-operator/pull/526/commits/b82b903f23d757345341e512d06f5349224b4736

We don't prefix any of the other patches so I didn't see why we should do it for this one.

```
❯ tree assets/csidriveroperators/*/hypershift -I generated
assets/csidriveroperators/aws-ebs/hypershift
├── guest
│   ├── claim_role.patch.yaml
│   ├── kustomization.yaml
│   └── monitoring_role.patch.yaml
└── mgmt
    ├── 02sa.patch.yaml
    ├── deployment.patch.yaml
    ├── hypershift_role.patch.yaml
    └── kustomization.yaml
assets/csidriveroperators/azure-disk/hypershift
├── guest
│   ├── claim_role.patch.yaml
│   ├── kustomization.yaml
│   └── monitoring_role.patch.yaml
└── mgmt
    ├── 03sa.patch.yaml
    ├── deployment.patch.yaml
    ├── hypershift_role.patch.yaml
    └── kustomization.yaml
assets/csidriveroperators/azure-file/hypershift
├── guest
│   ├── claim_role.patch.yaml
│   ├── kustomization.yaml
│   └── monitoring_role.patch.yaml
└── mgmt
    ├── 03sa.patch.yaml
    ├── deployment.patch.yaml
    ├── hypershift_role.patch.yaml
    └── kustomization.yaml
assets/csidriveroperators/openstack-cinder/hypershift
├── guest
│   ├── kustomization.yaml
│   └── monitoring_role.patch.yaml
└── mgmt
    ├── deployment.patch.yaml
    ├── hypershift_role.patch.yaml
    ├── kustomization.yaml
    └── sa.patch.yaml
assets/csidriveroperators/powervs-block/hypershift
├── guest
│   ├── 01_sa.yaml
│   ├── 02_role.yaml
│   ├── 03_rolebinding.yaml
│   ├── 04_clusterrole.yaml
│   ├── 05_clusterrolebinding.yaml
│   └── 07_cr.yaml
└── mgmt
    ├── 01_operator_role.yaml
    ├── 01_sa.yaml
    ├── 03_rolebinding.yaml
    └── 06_deployment.yaml
assets/csidriveroperators/shared-resource/hypershift
├── guest
│   └── 10_cr.yaml
└── mgmt
    ├── 02_sa.yaml
    └── 09_deployment.yaml

18 directories, 40 files
```

(note that `powervs-block` is a bit weird because they're not using `kustomize`. idk why but I wasn't going to make a potentially invasive change there to change that)